core/db/curibio/alembic/versions/be1504f90910_pulse_0_32_2.py

```diff
@@ -0,0 +1,24 @@
+"""pulse_0_32_2
```

Comment:
0.32.1 isn't on the prod cluster, so we should just replace all 0.32.1 stuff with 0.32.2. Make sure to double check the confluence doc for managing pulse3d versions in the cloud